requirements.txt

```diff
@@ -9,8 +9,8 @@
 # that are dependencies of libraries listed in the next section
 
 ga4gh-common==0.0.5
-ga4gh-schemas==0.0.8
-ga4gh-client==0.0.3
+ga4gh-schemas
```

Comment:
I agree with this. See my suggestion for referencing the constraints file in requirements file to pin the versions.